.github/CODEOWNERS

```diff
@@ -133,10 +133,13 @@
 /sdk/loganalytics/microsoft-azure-loganalytics/     @divyajay @alongafni
 
 # PRLabel:  %Monitor - Exporter
-/sdk/monitor/azure-monitor-opentelemetry-exporter   @trask @kryalama @ramthi @heyams
```

Comment:
@srnagar :  Is this removal from one of our partner teams?   I'm not sure if Ronnie and I need to adjust "Service Attention" bot rules to reflect this change.

Reply:
@jsquire yes, this is from one of our partner teams. The bot rules have to be updated too. If it's something I can update, please let me know.